src/games/mod.rs

```diff
@@ -92,15 +104,78 @@ pub mod unturned;
 pub mod ut;
 /// V Rising
 pub mod vr;
-/// Crysis Wars
-pub mod cw;
-/// Quake 2
-pub mod quake2;
-/// Quake 1
-pub mod quake1;
-/// Quake 3: Arena
-pub mod quake3a;
-/// Hell Let Loose
-pub mod hll;
-/// Soldier of Fortune 2
-pub mod sof2;
+
+use crate::protocols::gamespy::GameSpyVersion;
+use crate::protocols::minecraft::{JavaResponse, MinecraftVersion};
+use crate::protocols::quake::QuakeVersion;
+use crate::protocols::{self, Protocol};
+use crate::GDResult;
+use std::net::{IpAddr, SocketAddr};
+
+#[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
+#[derive(Debug, Clone, PartialEq)]
```

Comment:
I know that this is a draft, but just a heads up about remembering to import serde for these derives:
```rust
#[cfg(feature = "serde")]
use serde::{Deserialize, Serialize};
```